doc/toc.yml

```diff
@@ -65,7 +65,6 @@
       href: helpers/progress-extensions.md
     - name: Responsive markup extension
       href: helpers/responsive-extension.md
-    - name: StatusBar attached properties
```

Comment:
Why remove this?

Reply:
@kazo0 to fix this link that is not working as there is not related href and also it is a duplicate for StatusBar
![image](https://github.com/unoplatform/uno.toolkit.ui/assets/16295702/735157de-3bd5-4665-9353-4ec8a98ca7a3)